paseos/visualization/space_animation.py

```diff
@@ -0,0 +1,269 @@
+from paseos.actors.base_actor import BaseActor
+from paseos.actors.spacecraft_actor import SpacecraftActor
+from paseos.actors.ground_station_actor import GroundstationActor
+from paseos.paseos import PASEOS
+from paseos.visualization.animation import Animation
+
+import pykep as pk
+import numpy as np
+from dotmap import DotMap
+from typing import List
+import matplotlib.pyplot as plt
+from matplotlib.font_manager import FontProperties
+import matplotlib.animation as animation
+from matplotlib.artist import Artist
+
+
+class SpaceAnimation(Animation):
+    """This class visualizes the central body, local actor and known actors over time."""
+
+    fp = FontProperties(fname=r"./FontAwesome.otf")
```

Comment:
Is this required? If so, please move the file to the resources folder. Also, I think the path is still not relative as I get an error 
´[Errno 2] No such file or directory: '/home/pablo/ESA/PASEOS/my_notebooks/FontAwesome.otf'´

(note I am testing outside the module to find errors like this)